admin/Makefile

```diff
@@ -32,14 +32,34 @@ version-file: app/version.py
 Pipfile Pipfile.lock:
 	$(PIPENV) install
 
+# this is a hack because CircleCI env variables are awful
+ADMIN_SENTRY_DSN ?= $(ADMIN_SENTRY_DSN_DEV)
+ADMIN_SENTRY_ENV ?= $(ADMIN_SENTRY_ENV_DEV)
+
 build-frontend:
-	$(NPM) run build -- --production
+	ADMIN_SENTRY_DSN=${ADMIN_SENTRY_DSN} \
+	ADMIN_SENTRY_ENV=${ADMIN_SENTRY_ENV} \
+	  $(NPM) run build -- --production
+
+build-frontend-prod:
+	@$(MAKE)\
+	  ADMIN_SENTRY_DSN=${ADMIN_SENTRY_DSN_PROD} \
+	  ADMIN_SENTRY_ENV=${ADMIN_SENTRY_ENV_PROD} \
+	  build-frontend
 
-setup-frontend: ## Setup and build css
+install-frontend:
 	$(NPM) install
+
+setup-frontend: install-frontend ## Setup and build css
 	$(MAKE) build-frontend
 
-setup: setup-frontend version-file ## Setup virtualenv, install python deps, build css
+setup-frontend-prod: install-frontend ## Setup and build css
+	$(MAKE) build-frontend-prod
+
+setup: version-file ## Setup virtualenv, install python deps
+	$(PIPENV) install
+
+setup-prod: setup-frontend-prod version-file
```

Comment:
Could this be simplified with something like:
```
setup-prod: ADMIN_SENTRY_DSN=$(ADMIN_SENTRY_DSN_PROD)
setup-prod: ADMIN_SENTRY_ENV=$(ADMIN_SENTRY_ENV_PROD)
setup-prod: setup-frontend version-file
	$(PIPENV) install
```

and then remove `setup-frontend-prod` and `build-frontend-prod`